app/src/main/java/dadeindustries/game/gc/model/StellarPhenomenon/Sector.java

```diff
@@ -15,6 +17,8 @@
 	private int x;
 	private int y;
 
+	public ArrayList<Unit> ships = new ArrayList<Unit>();
```

Comment:
Does this present a problem for trying to compile a list of all a players ships, for things like calculating upkeep/special statuses?

Reply:
I don't think it causes any problem. This approach allows for efficient rendering and selection of ships within the UI. Basically, it only has to render the elements within the sectors that are currently visible on the screen. At the end of each turn, the whole global state has to be analysed anyway (moving ships from some sectors to others, creating spurious events, conflicts, etc) and that's when it's acceptable to have a bit of lag. If it's helpful, we can produce a list of current faction ships at the end of each turn. This wasn't meant to be a long comment, but say if you disagree.

Reply:
I agree with your argument.